dev/tools/gen_defaults/lib/list_tile_template.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'template.dart';
+
+class ListTileTemplate extends TokenTemplate {
+  const ListTileTemplate(super.blockName, super.fileName, super.tokens);
+
+  @override
+  String generate() => '''
+class _${blockName}DefaultsM3 extends ListTileThemeData {
+  const _${blockName}DefaultsM3(this.context)
+    : super(shape: ${shape("md.comp.list.list-item.container")});
+
+  final BuildContext context;
+
+  @override
+  Color? get tileColor => ${componentColor("md.comp.list.list-item.container")};
```

Comment:
shouldn't this be transparent? Tiles are already used on the scaffold, that has this color, but when the tile is not on top of the scaffold it's color needs to be set to transparent manually, so I don't see the point this is not directly transparent.

Reply:
Yes, this is being tracked here. https://github.com/flutter/flutter/issues/117700